multimodal/src/autogluon/multimodal/configs/optimization/adamw.yaml

```diff
@@ -16,7 +16,6 @@ optimization:
   skip_final_val: False # Flag to skip the last validation
   gradient_clip_val: 1
   gradient_clip_algorithm: "norm"
-  track_grad_norm: -1 # Whether to check gradient norm. We can set it to 2 to check for gradient norm.
```

Comment:
We still support using `track_grad_norm `, so no need to remove it. See https://github.com/autogluon/autogluon/blob/master/multimodal/src/autogluon/multimodal/optimization/lit_module.py#L401-L404